Competition-2020/src/main/cpp/CustomController.cpp

```diff
@@ -0,0 +1,72 @@
+#include <frc/WPILib.h>
+#include "CustomController.h"
+
+using namespace frc;
+
+CustomController::CustomController(int port) : XboxController(port) {
+    scale = 1; //Initialize the scale to 1.
+    square = false;
+}
+
+void CustomController::set_scale(float scale) {
```

Comment:
I don't see us using getters and setters, so you can leave that out if you want to (less work to do)